examples/kitchensink/webpack.config.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
+  entry: {
+    core: __dirname + '/core/index.js',
+    'app-color': __dirname + '/app-color/index.js',
+    'app-counter': __dirname + '/app-counter/index.js',
+    'app-reversed': __dirname + '/app-reversed/index.js',
+    'app-todos': __dirname + '/app-todos/index.js',
+    vendor: ['frint', 'frint-model', 'frint-react', 'frint-store', 'react', 'react-dom', 'rxjs']
```

Comment:
unless we add the `externals` key back again, all the vendors will also get included in the app bundles, increasing the size.